specs/agents/mobile/README.md

```diff
@@ -20,56 +20,56 @@ However, the Elastic mobile agents don't set every attribute defined (many only
+| [`device.manufacturer`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)     | `device.manufacturer` | `Apple`, `Samsung`            | :x: no  |  This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |   
 
 
 ### Common Span attributes
-| OTel Convention                         | Elastic Convention         | Example                          | Comment                            |
-|-----------------------------------------|----------------------------|----------------------------------| -----------------------------------|
-| [Instrumentation name](https://opentelemetry.io/docs/reference/specification/trace/api/#get-a-tracer) |`service.framework.name`| `SwiftUI`, `UIKit` ||
-| [Instrumentation version](https://opentelemetry.io/docs/reference/specification/trace/api/#get-a-tracer) |`service.framework.version`| `1.2.3`| |
-| [`net.host.connection.type`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.connection.type` | `cell`, `wifi` ||
-| [`net.host.connection.subtype`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.connection.subtype` | `lte`, `edge` ||
-| [`net.host.carrier.name`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.carrier.name` | [see OTEL](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) ||
-| [`net.host.carrier.mcc`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.carrier.mcc` | [see OTEL](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) ||
-| [`net.host.carrier.mnc`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.carrier.mnc` | [see OTEL](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) ||
-| [`net.host.carrier.icc`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) | `network.carrier.icc` | [see OTEL](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/) ||
+| OTel Convention                         | Elastic Convention         | Example                          | Required | Comment                            |
+|-----------------------------------------|----------------------------|----------------------------------| ---------| ---------------------------------|
+| [Instrumentation name / tracer name](https://opentelemetry.io/docs/reference/specification/trace/api/#get-a-tracer) |`service.framework.name`| `SwiftUI`, `UIKit` | :x: no | Use the name of the library that is instrumented.|
```

Comment:
Not sure how much this matters in that context, but there is a subtle difference: 
OTel's "Instrumentation name" refers to the instrumentation library (e.g. [opentelemetry-instrumentation-ioredis](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-ioredis)) while `service.framework.name` is the instrumented library (e.g. [ioredis](https://github.com/luin/ioredis)).
In case a library comes with built-in OTel, those libraries can be the same.

Reply:
Good point @z1c0 !

Currently, we (the APM server) map the OTel tracer / `Instrumentation name` to the internal field `service.framework.name` which is used to describe the target framework being instrumented.

Do you know if there's any other field in OTel that would allow us to set this information? I'm not aware of any resource attribute for this.

Reply:
Unfortunately, I'm not aware of an OTel field that would match `service.framework.name` either. There are tech-specific conventions like [`webengine.name`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/webengine.md) but nothing generic that comes to mind.

Another problem here is that resource attributes are probably not the right fit here, since there can be multiple instrumented frameworks per process.

A solution might be the [context-scoped attributes proposal](https://github.com/open-telemetry/oteps/pull/207) in combination with something like [source code attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#source-code-attributes) if these are extended with e.g. a "framework" or "library" aspect.